packages/sdk/src/Stream.ts

```diff
@@ -151,19 +137,15 @@ export class Stream {
     }
 
     /**
-     * Updates the metadata of the stream by merging with the existing metadata.
+     * Updates the metadata of a stream.
      */
     async update(metadata: Partial<StreamMetadata>): Promise<void> {
```

Comment:
Maybe add temporary merging behavior to `detectFields` (until we figure out can we remove that method or not)